### PR TITLE
Add meeting minutes for March 2023

### PR DESCRIPTION
**Pull request by tsyesika**
Here are the meeting minutes for the March 2023 meeting. I forgot to take note of who was attending so if you're missing or know of someone who's missing, please let me know and I'll add you.

### CODE REVIEW

**Comment by dckc** [comment]
I just noticed this by way of a notification of a comment on #32.

I'm used to sharing meeting notes causing a notification to participants. But as noted in [Process and goals](https://github.com/ocapn/ocapn/wiki/Process-and-goals)...

> No email list for participants [grumble]

It's important to keep the ball moving, so please tag one person from each of Agoric, Spritely, and capnproto as reviewers. Plus the chair.

File: meeting-minutes/2023-03-24.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  - Christine Lemmer Webber (cwebber)
+  - David Thompson (dthompson)
+  - Dan Connolly (dckc)
+  - Ian Denhardt (isd)
+  - Jessica Tallon (Jessica)
+  - Jonathan Rees (jar286)
+  - Kris Kowal (kriskowal)
+  - Mark Miller (markm)
+  - Juliana Sims (juliana)
+
```
Comment (ocdtrekkie) [inline meeting-minutes/2023-03-24.md:17]: - Jacob Weisz (ocdtrekkie)

I was silent, but present.

File: meeting-minutes/2023-03-24.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+jar286: going to create an issue about this topic
+
+# TOPIC: Agoric/Spritely interop status report Demonstrate Agoric & Spritely interop #6; next steps (@dckc)
+
+jar286: next topic is interoperability
+
+jar286: dckc has the floor
+
+dckc: this is a writeup from talking with Kris Kowal
```
Comment (dckc) [inline meeting-minutes/2023-03-24.md:108]:
```suggestion
dckc: [this demo sketch](https://github.com/ocapn/ocapn/issues/6#issuecomment-1477952429) is a writeup from talking with Kris Kowal and company
```

**Review by ocdtrekkie — CHANGES_REQUESTED** [review]

File: meeting-minutes/2023-03-24.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+Jessica: third one is the "OCapN URI" spec, maybe controversial now, represents sturdyrefs, how to establish in-bound and out-of-bound locations
+
+Jessica: that would be the three main documents, the fourth may not be included, but is the Syrup spec, which is the serialization form based on preserves
+
+Jessica: mainly written with a draft README, since the specs will *initially* based on Spritely's implementation, not to favor it, but as a starting point, but the group can take these draft specifications that we were writing
+
+Jessica: and then the group can evolve as the group reaches decisions
+
+Jessica: ETA on those, the Syrup one is ready for review, is ready as a PR on the syrup spec
```
Comment (dckc) [inline meeting-minutes/2023-03-24.md:44]: I believe we did note the URL of the PR during the meeting, so...

```suggestion
Jessica: ETA on those, the Syrup one is ready for review, is ready as [a PR on the syrup spec](https://github.com/ocapn/syrup/pull/9)
```

**Comment by ocdtrekkie** [comment]
> I just noticed this by way of a notification of a comment on #32.

I am "watching" this entire repository on GitHub. I receive an email notification on every new issue creation and comment repo-wide.

File: meeting-minutes/2023-03-24.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+jar286: can we get a concrete action item?
+
+jar286: going to create an issue about this topic
+
+# TOPIC: Agoric/Spritely interop status report Demonstrate Agoric & Spritely interop #6; next steps (@dckc)
+
+jar286: next topic is interoperability
+
+jar286: dckc has the floor
+
```
Comment (dckc) [inline meeting-minutes/2023-03-24.md:107]: let's skip stage direction such as this in the interest of conciseness

```suggestion
```

**Review by dckc — COMMENTED** [review]
oops... I somehow have 2 reviews in 2 tabs...

File: meeting-minutes/2023-03-24.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+dckc: this is a writeup from talking with Kris Kowal
+
+dckc: here spritely has a chat, spritely has spritely chat on one side, so spritely goes over a unix domain socket plugin which then the endo plugin for sockets, and then a captp bridge 
+
+dckc: and then a js chat app with an http endo plugin and a UI which goes on the Agoric side
+
+cwebber: I think this is an interesting goal towards trying to show off how this would work
+
+dckc: small amount of hacking I did do was about the coordinated types
```
Comment (dckc) [inline meeting-minutes/2023-03-24.md:116]:
```suggestion
dckc: [small amount of hacking](https://github.com/ocapn/ocapn/issues/5#issuecomment-1478965669) I did do was about the core data types
```

File: meeting-minutes/2023-03-24.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+isd: I think one of the questions is whether or not we can move those two things together
+
+isd: how attached are we from what parts of this data level
+
+isd: could you look at agoric's and see what parts we do and don't want to adopt?
+
+isd: I think having that conversation is the first order of businsess in terms of whether or not we can converge on a shared protocol
+
+markm: dckc said mapping a subset to a subset... if there's a joint subset that round trips, I'd describe that as a subset we've agreed upon.  so the layer separation we have now, that was partly inspired by one of these conversations.  I was trying to put collections in at the arrays/records stuff.  and I had to write down a lot of semantics of what it meant for two keys to be equal.  for example, if they're keys in a map or a set and
+
```
Comment (dckc) [inline meeting-minutes/2023-03-24.md:141]:
```suggestion

https://github.com/ocapn/ocapn/issues/5#issuecomment-1478684631 is a pre-meeting comment from markm to largely the same effect

```

File: meeting-minutes/2023-03-24.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+cwebber: tsyeika did an incredible job on this doc, encourage people to look at it
+
+dckc: if it's useful to have two people assigned, the interop demo is going to involve going from one thing to another, so I'm willing to be assigned as well
+
+jar286: 3 minutes left, I liked the traidtion we had in the w3c of doing issue review
+
+jar286: a drive to close issues or assign to people, so is there anything else we need to cover
+
+jar286: looks like Tuesday April 25 at 19:00 UTC (next meeting)
```
Comment (dckc) [inline meeting-minutes/2023-03-24.md:210]:
```suggestion
# Schedule for next monthly meeting

jar286: looks like Tuesday April 25 at 19:00 UTC (next meeting)
```

Hm... all but the 1st heading should be h2, i.e. `##`.

**Review by dckc — COMMENTED** [review]
I don't suppose any of these comments is stop-ship critical, but I do want to see how they're addressed before approving.

**Comment by jar398** [comment]
@dckc Yes, I'm watching "all activity".  Sometimes I get flooded and miss some things, but overall it hasn't been too burdensome.

**Comment by tsyesika** [comment]
Sorry this took me a while to get to, I've addressed all the feedback. Thanks!

**Review by ocdtrekkie — APPROVED** [review]

**Review by dckc — APPROVED** [review]

**Comment by tsyesika** [comment]
I think we can merge this now.

**Pull request merged**